content/ai/concepts/neural-networks/terms/stochastic-gradient-descent/stochastic-gradient-descent.md

```diff
@@ -0,0 +1,61 @@
+---
+Title: 'Stochastic Gradient Desent'
+Description: 'Stochastic Gradient Desent is optimizer algorithm that minimizes the loss functions in machine learning and deep learning models.'
+Subjects:
+  - 'Machine Learning'
+  - 'Deep Learning'
+  - 'Computer Science'
+Tags:
+  - 'AI'
+  - 'Neural Network'
+  - 'Optimizer'
+CatalogContent:
+  - 'paths/computer-science'
+  - 'paths/data-science'
+---
+
+**Stochastic Gradient Descent** (SGD) is a optimization algorithm. It is variant of gradient descent optimizer. The SGD minimize the loss function of machine learning algorithms and deep learning algorithms during backpropagation to update the weight and bias in Artificial Neural Networks. 
+
+The term stochastic mean randomness on which algorithm based upon. In this algorithm instead of taking whole dataset like grdient descent we take single randomly selected data point or small batch of data.suppose if the data set contains 500 rows SGD update the model parameters 500 times in one cycle or one epoch.
```

Comment:
```suggestion
The term `stochastic` means randomness on which the algorithm is based. In this algorithm, instead of taking whole datasets like `gradient descent`, we take single randomly selected data points or small batches of data. Suppose if the data set contains 500 rows SGD updates the model parameters 500 times in one cycle or one epoch.
```